### DISCUSSION TITLE
how to add another drive for records ?

### DISCUSSION THREAD

**Discussion by crazych1** [blakeblackshear/frigate#15657] [Ask A Question]
I have 2 hard drives, I have already connected one by redirecting /media/fragare/records, now I need to connect the second drive, and how do I get some of the cameras to write to another drive? How is it possible to do this? I've looked through all the documentation and haven't found anything suitable.

**Comment by NickM-27** [comment]
This can be done using a file system like merger.fs or fuse.fs

**Comment by crazych1** [comment]
It's very difficult, isn't there an easy way?

**Comment by banthungprong** [comment]
It is Linux, so there is only the normal way by configs. There are a lot of tutorials that might help.
the easy way: buy a bigger one.
